### DISCUSSION TITLE
Rename basic settings tab

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#1543] [UI Related]
There was a discussion about how to name the new tabs in settings.

@dcorson-ticino-com didn't like the term advanced/basic and I think it is a valid concern. What about renaming basic to audio/net?

**Comment by dcorson-ticino-com** [comment]
Sounds good to me.

**Reply by ann0see** [reply]
Ok. So the new name would be "Audio/Net" or "Audio/Network"?

**Reply by dcorson-ticino-com** [reply]
Audio/Network

**Reply by ann0see** [reply]
@dcorson-ticino-com can you open a PR with this change?

**Reply by dcorson-ticino-com** [reply]
Done

**Reply by ann0see** [reply]
Great! Thank you! (I don't see it yet, but I assume it will show up soon)

**Reply by ann0see** [reply]
Ok. I saw what you did: https://github.com/dcorson-ticino-com/jamulus/commit/4575fc22c12bba87d6547765463246bb75cb62db

Could you please update the branch or create a new one and then open a PR to the main repo?

**Reply by dcorson-ticino-com** [reply]
Should I also rename "User" to "User Setup" while I'm at it. ?
"User" seems lost on the screen alongside "Audio/Network Setup" and" Advanced Setup"

**Reply by ann0see** [reply]
That’s up to you.
